### PR TITLE
Added Romanian translation for index.

### PR DESCRIPTION
**Pull request by dragostis**
I started an endeavor to add a Romanian translation to the website. Since it's not that much to translate, I should manage by myself.

### CODE REVIEW

File: ro-RO/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          și oferă siguranță în cazul firelor de execuție.
+          <br/>
+          <b><a href="friends.html">Vezi cine folosește Rust.</a></b>
+        </p>
+      </div>
+      <div class="col-md-4">
+        <a class="release-button" href="install.html">
+          <div class="release-version">Instalează Rust <span>{{ site.stable }}</span></div>
+        </a>
+        <div class="release-date">{{ site.stable_date | date: "%B %-d, %Y" }}</div>
```
Comment (aznashwan) [inline ro-RO/index.html]: Shouldn't the date format have been "D/M/Y"?

**Comment by steveklabnik** [comment]
Thanks @dragostis ! Could you rebase this for me please?

**Comment by brson** [comment]
Needs another rebase 😭

**Comment by steveklabnik** [comment]
Yes, happy to merge this after a rebase.

**Comment by dragostis** [comment]
@steveklabnik I'd like to continue working on in until complete or at least close to it. I'm currently concentrating on the pest 1.0 release in my free time, but I should get back to it once that's done.

**Comment by steveklabnik** [comment]
@dragostis  I'm going to give this one a close for inactivity; if you want to update it, please let me know and I'm more than happy to re-open!

**Pull request closed without merging**